### ISSUE TITLE
ctrlp-py-matcher breaks fuzzy filtering in CtrlPBookmarkDir/Quickfix/Changes...

### ISSUE THREAD

**Issue by Konfekt** [FelikZ/ctrlp-py-matcher#15]
After `:CtrlPBookmarkDir<Cr>` you can enter text to do a fuzzy search of your bookmarked directories.

The tabulator space `  ` between the folder and the bookmark title becomes a litteral `\t` that `and joins the bookmark path and title.

This is on Linux, Vim 7.4.560

**Comment by Konfekt** [comment]
Would adding `\t` to 
```
 specialChars = ['^','$','.','{','}','(',')','[',']','\\','/','+']
```
in ctrlp-py-matcher.py help?

**Comment by Konfekt** [comment]
Note that I am using https://github.com/ctrlpvim/ctrlp.vim.

Perhaps commit `417ab865637acdc8f41c7ffa009c828bac028f54` is the culprit?

**Comment by FelikZ** [comment]
Hi @Konfekt ,

> Would adding \t to specialChars

I am not familar with `CtrlPBookmarkDir` logic, you could try it by your own and if it helps - I will merge it.

**Comment by Konfekt** [comment]
Ok. Did so and the issue persists. Works fine with cmatcher though. Perhaps compare with their code. They made changes to the new fork of CtrlP to make it work I think, see their commit 17ab865637acdc8f41c7ffa009c828bac028f54 at https://github.com/ctrlpvim/ctrlp.vim .

**Comment by FelikZ** [comment]
Please checkout the issue with HEAD version. Is probably fixed since #12 .

**Comment by Konfekt** [comment]
Thank you! I can confirm that it works fine.
